web-console/src/lib/compositions/streaming/import/useDefaultRows.ts

```diff
@@ -12,7 +12,7 @@ export const getDefaultValue = (columntype: ColumnType) =>
   match(columntype)
     .with({ type: 'BOOLEAN' }, () => false)
     .with({ type: 'TINYINT' }, { type: 'SMALLINT' }, { type: 'INTEGER' }, { type: 'BIGINT' }, () => 0)
```

Comment:
e.g., it looks like this is still a `number` type here even though BIGINT is a 64bit value that doesn't fit in a javascript number which is 64-bit floating point